dwave/optimization/include/dwave-optimization/nodes/mathematical.hpp

```diff
@@ -58,29 +58,37 @@ struct min {
 template <class T>
 struct modulus {};
 
-template <>  
```

Comment:
One thing that @arcondello is gonna mention is to not format unmodified code.

Reply:
To be clear, I am going to be a stickler and ask that these changes be undone. Please don't format code unrelated to the PR.

Reply:
I changed VScode to strip trailing whitespace because that's what NLSolver requires.  It's going be probably mean that I disable that and either manually add back in the whitespace or throw away this PR and create a new one.  Ultimately, it's your call but honestly it seems ridiculous to worry about trailing whitespace and not something that I've run across is 25 years of software development.

Reply:
It is ultimately a tradeoff between making PRs easy to digest vs tooling in IDEs. Either direction has pros and cons.